root/static/scripts/common/constants.js

```diff
@@ -80,6 +80,7 @@ export const FAVICON_CLASSES = {
   'bigcartel.com': 'bigcartel',
   'bookbrainz.org': 'bookbrainz',
   'books.apple.com': 'applebooks',
+  'boomkat': 'boomkat',
```

Comment:
```suggestion
  'boomkat.com': 'boomkat',
```
Although this whole system is a mess, but still better to follow the general style :)

Reply:
Sorry, missed that for some reason! Fixed...